mintlify/uma-connect/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Introduction"
+---
+import { umaaasProductName } from '/snippets/variables.mdx'
+
+
+With {umaaasProductName}, you can send and receive real-time UMA payments without touching crypto or implementing the UMA protocol. You onboard once, integrate a single API, and we route through the right licensed providers per region to handle Lightning, FX, and instant banking off-ramps.
+
+- **Single API, global reach**: Abstracts underlying providers (e.g., LSP, Triple-A, Bitso, Yellowcard) based on your locale.
```

Comment:
I'd get rid of the examples here. All that matters is the uma provider and we shouldn't think about grid switches in this context. Also we'll never use 3A.